WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -138,6 +138,14 @@ final class HubMenuViewModel: ObservableObject {
             }
         }
 
+        // Blaze menu.
+        // TODO: 9866 - async eligibility check
+        if featureFlagService.isFeatureFlagEnabled(.blaze), let index = generalElements.firstIndex(where: { item in
```

Comment:
Unless we want to skip adding the Blaze row if the payments row is not present, I think we should fall back to appending the Blaze row.

Reply:
Right now the Payments row is always visible, but I see your point we can append the row when the Payments row is conditionally hidden somehow in https://github.com/woocommerce/woocommerce-ios/pull/9868/commits/a36ba76f863a4c92b162e0212b669037f6bd3200. Also added some test cases for the menu elements to ensure the placement of the Blaze row in https://github.com/woocommerce/woocommerce-ios/pull/9868/commits/178d912606c6051709cbda487461406e2ba445d7.